ui/app/AppLayouts/Profile/stores/WalletStore.qml

```diff
@@ -162,8 +162,27 @@ QtObject {
         return networksModuleInst.getNetworkShortNames(chainIds)
     }
 
-    function copyToClipboard(textToCopy) {
-        globalUtils.copyToClipboard(textToCopy)
+    function processPreferredSharingNetworkToggle(preferredSharingNetworks, toggledNetwork) {
```

Comment:
Why is this added here? I thought we wouldn't add new stuff to a store (moreover so when it's a singleton)